script_test.go

```diff
@@ -1017,11 +1017,22 @@ func TestFindFiles_RecursesIntoSubdirectories(t *testing.T) {
 	}
 }
 
-func TestFindFiles_InNonexistentPathReturnsError(t *testing.T) {
+// FindFiles has been updated to ignore all errors
+// see [github issue #99]
+// (https://github.com/bitfield/script/issues/99)
+func TestFindFiles_FailsSilently(t *testing.T) {
```

Comment:
I think "fails silently" could be misinterpreted, couldn't it? Like "fails" suggests it's not working properly. What about something like:

```suggestion
func TestFindFiles_ReturnsEmptyPipeAndNoErrorForNonexistentPath(t *testing.T) {
```

But I'm not convinced this is the right behaviour, actually. I *am* convinced that if there's some error from `fs.WalkDir`'s traversal, we should ignore that. But if the user has asked us to `FindFiles` starting from some _root_ that doesn't exist, I think that _should_ be an error. Would you want to know that, as the programmer? I would.